plat/rpi/rpi5/platform.mk

```diff
@@ -99,9 +99,16 @@ BL31_SOURCES		+= drivers/scmi-msg/base.c			\
 				drivers/scmi-msg/entry.c		\
 				drivers/scmi-msg/smt.c			\
 				drivers/scmi-msg/reset_domain.c		\
+				drivers/scmi-msg/scmi_pinctrl.c		\
+				drivers/pinctrl/pinctrl.c		\
 				plat/rpi/rpi5/scmi/scmi.c		\
 				plat/rpi/rpi5/scmi/scmi_reset.c		\
-				plat/rpi/rpi5/rpi5_svc_setup.c
+				plat/rpi/rpi5/rpi5_svc_setup.c \
+				plat/rpi/rpi5/bcm2712_pinctrl.c
```

Comment:
Probably it's better to split these changes between commts that are adding protocol and pinctrl driver? What do you tihnk?

Reply:
Not sure it make sense as pinctrl feature should be enabled all together.

Reply:
updated